arrows/vxl/hashed_image_classifier.cxx

```diff
@@ -0,0 +1,498 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "hashed_image_classifier.h"
+
+#include <vital/range/iota.h>
+
+#include <vil/vil_plane.h>
+
+#include <cmath>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace vxl {
+
+// ----------------------------------------------------------------------------
+// Classify a chain of hashed feature images
+template < typename FeatureType, typename OutputType >
+void
+hashed_image_classifier< FeatureType, OutputType >
+::classify_images( input_image_t const& input_features,
+                   weight_image_t& output_image,
+                   weight_t const offset ) const
+{
+  feature_vector_t input_features_vector;
+  for( auto const i : vital::range::iota( input_features.nplanes() ) )
```

Comment:
Not really a suggestion, but more just an FYI; if you get tired of typing all this, I often alias `namespace kvr = kwiver::vital::range`.